routes/activities.js

```diff
@@ -1,19 +1,43 @@
-const express = require('express');
-const router = express.Router();
-const {upload} = require('../S3-sdkConfig');
+const { Router } = require('express')
+const router = Router()
+const { body, check } = require('express-validator')
+const activitiesController = require('../controllers/activities')
+const { validateFields } = require('../middlewares/validateFields')
+const { upload } = require('../S3-sdkConfig')
+const controller = require('../controllers/activities')
 
-const controller = require('../controllers/activities');
+const isRequired = prop => {
```

Comment:
Si es por reutilizar el is require, no vale la pena este metodo de ultima crea una constante que tenga is require y concatena los string

Reply:
no entendí